### PR TITLE
M03_L08 small textual changes

### LINKED ISSUE #600: M03-L08_AZ400_Control_Deployments_using_Release_Gates: Sync with Azure portal
### Contact Details

maarten.vandiemen@infosupport.com

### What happened?

Exercise : 02
Task: 02
Step: 03

Description of issue: classic is removed from the Azure UI

Exercise : 02
Task: 02
Step: 08

Description of issue: Application Insights is moved to _Monitoring_ instead of _Settings_.

Exercise : 02
Task: 02
Step: 14

Description of issue: UI has changed for creating a rule.

Exercise: 02
Task 01
Step: 5

Description of issue: S1 is deprecated

### Lab

Control Deployments using Release Gates

### Relevant screenshots

Task 2 - step 3 (current UI in Azure Portal)
image

Task 1 - step 5
image

Task 2 - step 8 (current UI in Azure Portal)
image

Task 2 - step 14 (current UI in Azure Portal)
image

### Do you want to help us? 👏

- [X] Create a Pull Request by following this project's [Contribution Guide](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#how-can-i-collaborate)

### PR DESCRIPTION
**Pull request by maartenvandiemen**
## Related Issue

**Link related Github Issue** 🢂 Fixes #600.

## Checklist 
Mark completed with "x" between brackets, "[x]", or checking the box once the PR is created:
- [X] Has related GitHub Issue 💥 [Create Issue](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#reporting-issues) 📝
- [X] Tested it
- [X] Read the PR collaboration guide 👓 [Collaboration Guide](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#pull-requests) 📝

Changes proposed in this pull request:

- Small textual changes based on the latest changes in the Azure Portal

### CODE REVIEW

File: Instructions/Labs/AZ400_M03_L08_Control_Deployments_using_Release_Gates.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ In this task, you will create two Azure web apps representing the **DevTest** an
 
    ```bash
    SERVICEPLANNAME='az400m04l09-sp1'
-   az appservice plan create -g $RESOURCEGROUPNAME -n $SERVICEPLANNAME --sku S1
+   az appservice plan create -g $RESOURCEGROUPNAME -n $SERVICEPLANNAME --sku P0V3
```
Comment (rob-foulkrod) [inline Instructions/Labs/AZ400_M03_L08_Control_Deployments_using_Release_Gates.md:137]: I'd like to hold off on changing SKUs until there is some stability on Azure Pass.

**Review by rob-foulkrod — COMMENTED** [review]
Appreciate the updates, if you could revert the SKU change we can bring this in.

File: Instructions/Labs/AZ400_M03_L08_Control_Deployments_using_Release_Gates.md (hunk 1 above)
Comment (yashints) [reply Instructions/Labs/AZ400_M03_L08_Control_Deployments_using_Release_Gates.md:137]: If we're to move on from Azure Pass this won't even be possible, doing down might be a better option with B1 or B2

Comment (maartenvandiemen) [reply Instructions/Labs/AZ400_M03_L08_Control_Deployments_using_Release_Gates.md]: Based on the comments above, I have reverted this change. Let's see how the changes to the Azure Pass work out for this.

**Comment by maartenvandiemen** [comment]
> Appreciate the updates, if you could revert the SKU change we can bring this in.

Fixed

**Comment by LuizMacedo** [comment]
Thank you @maartenvandiemen!

**Review by LuizMacedo — APPROVED** [review]

**Pull request merged**